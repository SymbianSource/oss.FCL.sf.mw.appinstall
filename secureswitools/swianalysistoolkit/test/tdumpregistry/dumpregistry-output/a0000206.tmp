Registry Controller Information:

Registry Entry :.\data\a0000206\00000000.reg

Package UID :0xa0000206
Package Name :3 Data Files
Vendor :Symbian
Package Index :0

Controller1 Version :2.5.4
Controller1 Algorithm :SHA1

Number of Dependencies : 0

Number of Embedded Packages : 0

SISProperty Count:0

Number Of File Descriptions :3

SIS File Description:

Target1:c:\restore\pic1.bmp
File Operation Type :Install
Algorithm :SHA1
SID :0x0

Target2:c:\restore\pic2.bmp
File Operation Type :Install
Algorithm :SHA1
SID :0x0

Target3:c:\restore\pic3.gif
File Operation Type :Install
Algorithm :SHA1
SID :0x0

Validation Status : ValidatedToAnchor
Revocation Status : OcspNotPerformed
