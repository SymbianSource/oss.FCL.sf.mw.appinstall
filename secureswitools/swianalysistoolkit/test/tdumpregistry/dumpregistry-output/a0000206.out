Package Information:

Registry File :.\data\a0000206\00000000.reg

Package UID :0xa0000206
Package Name :3 Data Files
Vendor :Symbian
Package Index :0
Drives :4
No of executables :0
Package Version :2.5.4
Languages :English
Selected Drive :No Drive Selected
File Major Version :4
File Minor Version :0
Package Vendor Localized Name :Symbian
Package Install Type :Installation
Is Package in ROM :No
Package Preinstalled :No
Package Signed :Yes
Package Trust :SisPackageCertificateChainValidatedToTrustAnchor
